### PR TITLE
windows: Add tools support with batch scripts.

### PR DESCRIPTION
**Pull request by priyank-p**
Add windows support.

### CODE REVIEW

File: tools/reinstall-node-modules.cmd [hunk 1]
```diff
@@ -0,0 +1,3 @@
+deltree /y node_modules
```
Comment (akashnimare) [inline tools/reinstall-node-modules.cmd]: This doesn't work. Use `rmdir /s /q`.

**Review by akashnimare — CHANGES_REQUESTED** [review]

File: tools/reinstall-node-modules.cmd (hunk 1 above)
Comment (priyank-p) [reply tools/reinstall-node-modules.cmd]: I fixed it. BTW do other script works?

**Review by akashnimare — APPROVED** [review]

**Comment by priyank-p** [comment]
Here is how to use this in windows:
Regular old cmd prompt: `.\tools\fetch-pull-request` no need to have .cmd at the end.
Git Bash and powershell: `./tools/fetch-pull-request` just like unix

File: tools/reinstall-node-modules.cmd [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@echo off
 rmdir /s /q node_modules
```
Comment (akashnimare) [inline tools/reinstall-node-modules.cmd]: @cPhost somewhere here.

File: tools/reinstall-node-modules.cmd [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@echo off
+
+echo "Removing node_modules and app/node_modules"
+rmdir /s /q node_modules
+rmdir /s /q app/node_modules
+
+echo "node_modules removed reinstlling npm packages"
```
Comment (akashnimare) [inline tools/reinstall-node-modules.cmd]: Typo - **reinstalling**

File: tools/reinstall-node-modules [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+set -x
+
+echo "Removing node_modules and app/node_modules"
+rm -rf node_modules
+rm -rf app/node_modules
+
+echo "node_modules removed reinstlling npm packages"
```
Comment (akashnimare) [inline tools/reinstall-node-modules]: Typo - **reinstalling**

**Review by akashnimare — CHANGES_REQUESTED** [review]

**Review by akashnimare — APPROVED** [review]

**Comment by akashnimare** [comment]
Merged, thanks @cPhost 🚀

**Pull request merged**